FIPS/config.md

```diff
@@ -0,0 +1,499 @@
+; Sub-path relative to HomeDir to store Ldb files
+;dbLdbPath = database/ldb
+
+; Sub-path relative to HomeDir for BoltDB
+;dbBoltPath = database/bolt
+
+; If enabled, factomd will turn on the block extractor to export blocks to disk
+;dbExportData = false
+
+; Sub-path relative to HomeDir for exporting data
+;dbExportDataPath = database/export/
+
+; Sub-path relative to HomeDir for the block extractor
+;dbDataStorePath = data/export
+
+; Disable the use of the FastBoot file to cache block validation
+;dbNoFastBoot = false
+
+; Create a FastBoot entry every X blocks
+;dbFastBootRate = 1000
+
+; ---------------- P2P ----------------
+; Disable the peer to peer network
+;p2pDisable = false
+
+; The filename suffix of the peers file which is added to the current network
+;p2pPeerFileSuffix = "peers.json"
+
+; The default port used for network connections
+;p2pPort = 8108
+
+; The URL of the seed file to use for bootstrapping
+;p2pSeed =
+
+; How many peers to broadcast messages to
+;p2pFanout = 16
+
+; A comma-separated list of peers that the node will always connect to in the format of "host:port"
+; Example to add four special peers:
+;   p2pSpecialPeers = "123.456.78.9:8108,97.86.54.32:8108,56.78.91.23:8108,hostname:8108"
+;p2pSpecialPeers = 
+
+; Which peers the node should allow.
+; Choices:
+;   NORMAL: allows all connections (default)
+;   ACCEPT: the node accepts incoming connection but only dials to special peers
+;   REFUSE: the node dials to special peers but refuses all incoming connections
+;p2pConnectionPolicy = NORMAL
+
+; How long peers have to send or receive a message before timing out
+;p2pTimeout = 5m
+
+
+; ---------------- LOGGING ----------------
+; The level of messages to log. Setting includes all options to the right
+; Choices:
+;   DEBUG | INFO | NOTICE | WARNING | ERROR | CRITICAL | ALERT | EMERGENCY | NONE
+;logLevel = ERROR
+
+; The sub-path of HomeDir to store logs in
+;logPath = database/Log
+
+; If enabled, log files will be written in JSON
+;logJson = false
+
+; The URL of a logstash server to send logs to. Leave blank to disable
+;logLogstash = 
+
+; Specify a file to write a copy of StdOut to file
+;logStdOut =
+
+; Specify a file to write a copy of StdErr to file
+;logStdErr = 
+
+; A regular expression of which message logs to save in the current working directory
+; For more details see https://factomize.com/forums/threads/logging-in-factomd.1766/
+;logMessages = 
+
+; Save DBStates to disk after being processed
+; Files will be saved to dbLdbPath/<network>/dbstates/processed_dbstate_<height>.block
+;logDBStates = false
+
+
+; ---------------- SIMULATION ----------------
+; Disable keyboard input to the console
+;simNoInput = false
+
+; How many simulated nodes to launch with a minimum of one
+;simCount = 1
+
+; The node to focus on at startup. The first node starts at 0
+;simFocus = 0
+
+; The network structure of the simulated network
+; Choices: FILE | SQUARE | LONG | LOOPS | ALOT | ALOT+ | TREE | CIRCLES
```

Comment:
This is a really minor comment, but we should default to `ALOT+`. If you run a long network, and do election testing locally, it is very easy to accidentally split the network. We've run into issues in the past, where we think it is a code failure, but it was really just the network topology being segmented.

Reply:
sounds good to me